flang/docs/ParserCombinators.md

```diff
@@ -141,7 +141,7 @@ collect the values that they return.
 * `applyLambda([](&&x){}, p1, p2, ...)` is the same thing, but for lambdas
   and other function objects.
 * `applyMem(mf, p1, p2, ...)` is the same thing, but invokes a member
-  function of the result of the first parser for updates in place.
+  function of the result of the first parser.
```

Comment:
Does the new version no longer work for updates in place?

Reply:
It returns the value returned by the member call, so for member functions returning void (as it was before) it wouldn't work.  For member functions returning reference to `*this` or something equivalent it should be fine.

Is modifying in-place is still needed?  If so I'd need to separate the code from this PR into its own parser.

Reply:
You'll need to look at usage sites (of which there are not many) to be sure this comment is obsolete.  I wouldn't have added that comment unless I had been thinking about mutation in situ at the time.

Reply:
I couldn't find any uses of this in the upstream sources.  I thought that maybe it's used in some downstream project.

Reply:
Go for it, then.  Downstream usage can adapt if needed.